back/app/services/idea_custom_fields_service.rb

```diff
@@ -40,13 +40,13 @@ def submittable_fields_with_other_options
 
   # Used in the printable PDF export
   def printable_fields
-    ignore_field_types = %w[section page date files image_files point file_upload shapefile_upload topic_ids cosponsor_ids ranking matrix_linear_scale]
+    ignore_field_types = %w[section page date files image_files point file_upload shapefile_upload topic_ids cosponsor_ids ranking matrix_linear_scale sentiment_linear_scale]
     fields = enabled_fields.reject { |field| ignore_field_types.include? field.input_type }
```

Comment:
I was assuming we'd support this (Formsync), so I added it. Let me know if I shouldn't though.